src/components/FormElement.tsx

```diff
@@ -1,14 +1,25 @@
 import type {ForwardedRef} from 'react';
-import React, {forwardRef} from 'react';
+import React, {forwardRef, useRef} from 'react';
 import type {ViewProps} from 'react-native';
 import {View} from 'react-native';
+import useFormSetup from '@hooks/useFormSetup';
 import * as ComponentUtils from '@libs/ComponentUtils';
 
-function FormElement(props: ViewProps, ref: ForwardedRef<View>) {
+function FormElement(props: ViewProps, outerRef: ForwardedRef<View>) {
+    const formRef = useRef<View | null>(null);
+    useFormSetup(formRef.current);
```

Comment:
Would you add a new line before and after this call? This is likely subjective, but I believe this important call will stand out better.